README.md

```diff
@@ -1,71 +1,45 @@
-# ocm-vscode-extension README
+# OCM VScode Extension README
 
-This is the README for your extension "ocm-vscode-extension". After writing up a brief description, we recommend including the following sections.
+ "ocm-vs-extension" helps creating OCM cluster applications. 
 
-## Features
+## Current Features
 
-- init clusters
-- join clusters
-- OCM crd snippest
-- create OCM project
+- Loading OCM crd snippets from the command pallette OCM 
 
-\!\[feature X\]\(images/feature-x.png\)
+![loading OCM crd snippets from the command pallete OCM](images/snippets-from-pallete.gif)
 
-> Tip: Many popular extensions utilize animations. This is an excellent way to show off your extension! We recommend short, focused animations that are easy to follow.
+- Create a sample Git-type Application Lifecycle Project
 
-## Requirements
-
-If you have any requirements or dependencies, add a section describing those and how to install and configure them.
-
-## Extension Settings
+![Create a sample Git-type Application Lifecycle Project](images/new-project.gif)
 
-Include if your extension adds any VS Code settings through the `contributes.configuration` extension point.
 
-For example:
+## Requirements
+ 
+ 
+ - To apply generated resources the [Application Lifecycle Management Addon][0] should be installed and enabled in your hub/managed clusters.
 
-This extension contributes the following settings:
 
-* `myExtension.enable`: enable/disable this extension
-* `myExtension.thing`: set to `blah` to do something
 
 ## Known Issues
 
 Calling out known issues can help limit users opening duplicate issues against your extension.
 
 ## Release Notes
 
-Users appreciate release notes as you update your extension.
-
-### 1.0.0
-
-Initial release of ...
-
-### 1.0.1
-
-Fixed issue #.
-
-### 1.1.0
-
-Added features X, Y, and Z.
-
------------------------------------------------------------------------------------------------------------
-## Following extension guidelines
+### 1.0.0  
```

Comment:
I'm not sure we need this section at all.
If we're going to leverage GitHub releases,
they have a great feature for auto-generating the release notes.
maintaining a designated section in the README.md file seems redundant.